src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/CCJStaffNotificationService.java

```diff
@@ -69,7 +69,7 @@ private EmailData prepareEmailData(final Claim claim) {
         map.put("claimReferenceNumber", claim.getReferenceNumber());
         map.put("claimantName", claim.getClaimData().getClaimant().getName());
         map.put("claimantType", PartyUtils.getType(claim.getClaimData().getClaimant()));
-        map.put("defendantName", claim.getClaimData().getDefendant().getName());
+        map.put("defendantName", claim.getCountyCourtJudgment().getDefendant().getName());
```

Comment:
I think the same should be done in `generateCcjPdf` method below (line 87).

Reply:
you can't change the name on the CCJ though?

Reply:
Nope you cant change name or email address on CCJ

Reply:
so what is this PR actually doing then?

Reply:
Agree with Rafal, to remove confusion I would like to take all details from CCJ even if they are the same as the claim one.

Reply:
There's a small question of whether they should be on the CCJ at all.

Reply:
Probably not if they are not modifiable :)

Reply:
anyway I have changed that